frontend/build.go

```diff
@@ -117,7 +117,13 @@ func BuildWithPlatform(ctx context.Context, client gwclient.Client, f PlatformBu
 
 // Like [BuildWithPlatform] but with a pre-initialized dockerui.Client
 func BuildWithPlatformFromUIClient(ctx context.Context, client gwclient.Client, dc *dockerui.Client, f PlatformBuildFunc) (*gwclient.Result, error) {
-	rb, err := dc.Build(ctx, func(ctx context.Context, platform *ocispecs.Platform, idx int) (gwclient.Reference, *dalec.DockerImageSpec, *dalec.DockerImageSpec, error) {
+	rb, err := dc.Build(ctx, func(ctx context.Context, platform *ocispecs.Platform, idx int) (_ gwclient.Reference, _ *dalec.DockerImageSpec, _ *dalec.DockerImageSpec, retErr error) {
+		defer func() {
+			if r := recover(); r != nil {
+				retErr = fmt.Errorf("recovered panic in build: %+v", r)
```

Comment:
Is there ever a case where `retErr` could be non-nil when this deferred func runs?
```
				retErr = errors.Join(retErr, fmt.Errorf("recovered panic in build: %+v", r))
```

Reply:
Sorry... I said that wrong... if there's a panic, then it would always be nil.